content/concepts/websockets.textile

```diff
@@ -427,7 +427,7 @@ h3(#socketio). Socket.io
 
 "Socket.io":https://github.com/socketio/socket.io has been around for a while now and could be thought of as the "jQuery" of WebSockets. It uses "long polling":/concepts/long-polling and WebSockets for its transports, by default starting with "long polling":/concepts/long-polling and then upgrading to WebSockets if available. Given the waning relevance of "long polling":/concepts/long-polling, Socket.io's main drawcards these days are its "other features":https://stackoverflow.com/questions/38546496/moving-from-socket-io-to-raw-websockets/38546537#38546537, such as restoring dropped connections, automatic support for JSON, and "namespaces", which are essentially isolated messaging channels multiplexed over the same client connection.
 
-Socket.io isn't actually interchangeable with generic WebSockets solutions - either on the server side or on the client side - and attempting to connect to something other than a Socket.io client or server will fail. It has its own additional handshake protocol, and some additional metadata included in each message.
+"Socket.io":https://www.ably.io/concepts/socketio isn't actually interchangeable with generic WebSockets solutions - either on the server side or on the client side - and attempting to connect to something other than a Socket.io client or server will fail. It has its own additional handshake protocol, and some additional metadata included in each message.
 
 Should you use it? On the plus side, in addition to "the simplicity of getting up and running":https://socket.io/get-started/chat/, it's well-established and there is a wealth of learning material out there if you get stuck. On the other hand, there have been plenty of reports of memory leaks and general performance issues over time, and despite there being "a mountain of issues":https://github.com/socketio/socket.io/issues in the Github repository, responses are few, and "commits and updates":https://github.com/socketio/socket.io/commits/master these days seem relatively infrequent. Just like jQuery, Socket.io is largely a product of a bygone era, and for new projects, you'd be better off with something a little more modern. jQuery does still have its fans though, so...
```

Comment:
Can you please replace the strike through links with the links in code wrap:

"the simplicity of getting up and running":~https://socket.io/get-started/chat/~ `https://ably.com/concepts/socketio`

"a mountain of issues":~https://github.com/socketio/socket.io/issues~ `https://www.ably.io/concepts/socketio#future`

"commits and updates":~https://github.com/socketio/socket.io/commits/master~ `https://www.ably.io/concepts/socketio#future`